modules/openapi-generator/src/main/resources/typescript-fetch/runtime.mustache

```diff
@@ -148,7 +148,8 @@ export class BaseAPI {
             body:
                 isFormData(overridedInit.body) ||
                 overridedInit.body instanceof URLSearchParams ||
-                isBlob(overridedInit.body)
+                isBlob(overridedInit.body) ||
+                !isJsonMime(context.headers['Content-Type'])
```

Comment:
I think we should treat undefined as json here, just to be compatible with the current behavior

Reply:
done :)

Reply:
please add the undefined check here instead of checking it inside the isJsonMime